rholang/src/main/scala/coop/rchain/rholang/interpreter/Reduce.scala

```diff
@@ -1334,6 +1334,64 @@ class DebruijnInterpreter[M[_], F[_]](
       } yield result
   }
 
+  private[this] val toSet: Method = new Method() {
+    def toSet(baseExpr: Expr): M[Par] =
+      baseExpr.exprInstance match {
+        case e: ESetBody =>
+          (e: Par).pure[M]
+        case EListBody(EList(basePs, locallyFree, connectiveUsed, remainder)) =>
+          (ESetBody(
+            ParSet(
+              basePs,
+              connectiveUsed,
+              locallyFree.get.pure[Coeval],
+              remainder
+            )
+          ): Par).pure[M]
+        case other =>
```

Comment:
I'm curious what your reasoning is for not implementing `toSet` and `toMap` for all Rholang collections that are scala collections under-the-hood. You should just be able to delegate down to the scala collection converters.